lib/server.js

```diff
@@ -73,6 +73,13 @@ function createServer (options, httpHandler) {
       }
     }
 
+    try {
+      Object.defineProperty(this, 'listeningOrigin', {
+        value: `${options.https ? 'https' : 'http'}://${host}:${listenOptions.port}`,
+        writable: false
+      })
+    } catch (e) {}
```

Comment:
How is it possible for this to throw?

Reply:
Some tests fail without try/catch, for example:

<img width="365" alt="image" src="https://user-images.githubusercontent.com/4263614/230714243-766203f0-5465-4097-a791-3d7e53f9b836.png">

I will investigate to better understand the issue

Reply:
This seems more like a side-effect of the change rather than a possible "failure" scenario.
Remember that Fastify listens, by default, to the two loopback hosts (`127.0.0.1` and `::1`), maybe the issue goes in that regard.